src/nxt_openssl.c

```diff
@@ -14,6 +14,10 @@
 #include <openssl/bio.h>
 #include <openssl/evp.h>
 
+#if OPENSSL_VERSION_NUMBER >= 0x30000000L
+/* TODO removed when SSL_CTX_set_tlsext_ticket_key_cb updated to SSL_CTX_set_tlsext_ticket_key_evp_cb */
```

Comment:
What needs to be done exactly?

Is the replacement difficult?

Reply:
I'd prefer adding -Wno-error=deprecated-declarations to the build system, so that we get a reminder every time we compile that the warning is to be fixed.

Reply:
We don't currently get the OpenSSL 3.0 deprecation warnings due to https://github.com/nginx/unit/commit/5665838b680cdc06d6eb83107e4d46db6cfae0c4